shaders/landscape.frag

```diff
@@ -0,0 +1,22 @@
+#version 320 es
+#define M_PI 3.1415
+precision mediump float;
+
+uniform vec4 color_base1;
+uniform vec4 color_base2;
+uniform vec4 color_stripe;
+// uniform float freq;
+// uniform float threshold;
```

Comment:
Isn't it impossible to use these variables instead of `in_vec`?

Reply:
@Aki-7 
I tried but it seemed to me that `zukou::BaseTechnique::Uniform()` only accepts `glm::mat` arguments. Maybe I'm wrong.
I thought using `glm::vec<1, float>` would be a workaround, but there's no `vec1` in GLSL so I decided to use `vec2` instead.

Reply:
You can use `glm::vec<1, float>` for `float` in GLSL. Sorry for less documentation.

Reply:
Oh ok! Will try out!